core/modules/user/user.module

```diff
@@ -3219,6 +3314,10 @@ function user_register_page() {
 function user_register_form($form, &$form_state) {
   global $user;
   $admin = user_access('administer users');
+  $user_login_appearance_setting = user_login_appearance_setting();
+  if ($user_login_appearance_setting === 'simplified') {
+    $form['#theme_wrappers'] = array('form', 'user_simplified_page');
+  }
```

Comment:
In order to avoid the problem highlighted in this comment, it's best to use `user_page_is_simplified_page()` to check if this form should be rendered as simplified.

 ```suggestion
    if (user_page_is_simplified_page()) {
      $form['#theme_wrappers'] = array('form', 'user_simplified_page');
    }
```